front/app/api/project_folder_files/keys.ts

```diff
@@ -0,0 +1,17 @@
+import { QueryKeys } from 'utils/cl-react-query/types';
+import { IQueryParameters } from './types';
+
+const baseKey = {
+  type: 'project_folder_file',
```

Comment:
Are we sure this is the correct type? My guess would be that this is type "file" and we need to add a variant "folder" to it.